website/src/docs/uppy.md

```diff
@@ -751,16 +751,6 @@ uppy.on('file-removed', (file, reason) => {
 })
 ```
 
-```js
-uppy.on('file-removed', (file, reason) => {
-  removeFileFromUploadingCounterUI(file)
-
-  if (reason === 'removed-by-user') {
-    sendDeleteRequestForFile(file)
-  }
-})
-```
-
```

Comment:
What's the reason for removing this? It looks unrelated.

Reply:
This should not have been committed here, but I wanted to remove this redundant example based on #2988